src/ParameterDistributions.jl

```diff
@@ -440,7 +449,10 @@ end
 Apply the transformation to map parameter samples `xarray` from the unconstrained space into (possibly constrained) space.
 Here, `xarray` contains parameters as columns and samples as rows.
 """
-function transform_unconstrained_to_constrained(pd::ParameterDistribution, xarray::Array{FT, 2}) where {FT <: Real}
+function transform_unconstrained_to_constrained(
+    pd::ParameterDistribution,
+    xarray::AbstractMatrix{FT},
```

Comment:
I would leave this untyped as well

Reply:
@jakebolewski I can leave the one above untyped, but I think I need to specify the type on this one because we are using the type for dispatch. We have three `transform_unconstrained_to_constrained`, and I presume only one of them can be the default untyped version.

Reply:
ah ok, you are correct (didn't see that from the diff)